plugins/main/bridge/bridge.go

```diff
@@ -358,7 +358,6 @@ func setupVeth(netns ns.NetNS, br *netlink.Bridge, ifName string, mtu int, hairp
 	if err != nil {
```

Comment:
Can you remove this block of code since it's now superfluous?

Reply:
This turned out to be something we were doing. I am closing this.